### DISCUSSION TITLE
After forgot password link click automatic login without showing new password page.

### DISCUSSION THREAD

**Discussion by Keval-R** [supabase/supabase#13994] [Questions] [answered]
I follow next js  + superbase.(Link:https://supabase.com/docs/guides/getting-started/tutorials/with-nextjs)

- I face an issue with auto login after clicking the reset password link. So anyway to stop autologin?
- I follow @supabase/auth-ui-react plugin for authentication.

**Comment by silentworks** [comment, marked as answer]
There is currently no way to stop the auto-login. This is used as part of the experience where it logs you in and takes you to the reset password page. Unfortunately the example guide you posted doesn't cover password reset at the moment. This is something we are working on improving at the moment and should have some updates in the coming weeks.
